modules/test/playwright/helpers/HeadlessAdminUserApiHelper.ts

```diff
@@ -187,6 +200,18 @@ export class HeadlessAdminUserApiHelper {
 		);
 	}
 
+	async postAccountUserAccountByEmailAddress(
+		accountId: number,
+		accountRoleIds: number[],
+		emailAddresses: string[]
+	) {
+		return this.apiHelpers.post(
+			`${this.apiHelpers.baseUrl}${this.basePath}/accounts/${accountId}/user-accounts/by-email-address` +
+				(accountRoleIds ? `?accountRoleIds=${accountRoleIds}` : ''),
```

Comment:
`${this.apiHelpers.baseUrl}${this.basePath}/accounts/${accountId}/user-accounts/by-email-address${(accountRoleIds ? `?accountRoleIds=${accountRoleIds}` : '')}`